### PR TITLE
Updating look & feel

### PR DESCRIPTION
**Pull request by glauco**
Matching example colors with AGCO colors.

### CODE REVIEW

File: app/styles/main.css [hunk 1]
```diff
@@ -8841,53 +8841,80 @@ body {
   margin: 30px 0;
 }
 
+/* line 50, main.scss */
+.panel-primary {
+  border-color: #AE132A;
```
Comment (waldemarnt) [inline app/styles/main.css]: 👍

File: app/styles/main.css [hunk 2]
```diff
@@ -8841,53 +8841,80 @@ body {
...
+}
+
+/* line 54, main.scss */
+.navbar-default {
+  background-color: #AE132A;
```
Comment (waldemarnt) [inline app/styles/main.css]: 👍 agco color

**Comment by jeffhsta** [comment]
:+1:

**Comment by waldemarnt** [comment]
👍

**Pull request merged**